base/docs/Docs.jl

```diff
@@ -676,14 +676,15 @@ end
 
 
 """
-    check_documented(mod::Module; all=false)
+    undocumented_names(mod::Module; all=false)
 
-Check all names in the module are documented. `all` determines which names are checked,
-following the behavior of `names(mod; all)`.
+Return the undocumented names in `module`. `all=false` returns only public names;
+`all=true` also includes nonpublic names, following the behavior of [`names`](@ref).
+
+See also: [`names`](@ref), [`Docs.hasdoc`](@ref).
 """
-function check_documented(mod::Module; all=false)
-    nodocs = filter!(sym -> !hasdoc(mod, sym), names(mod; all))
-    isempty(nodocs) || error("missing docstrings in $mod: $nodocs")
+function undocumented_names(mod::Module; all=false)
+    filter(!hasdoc, names(mod; all))
 end
```

Comment:
```suggestion
undocumented_names(mod::Module; all=false) = filter(!hasdoc, names(mod; all))
```

Reply:
That doesn't work because `names` returns `Symbols` not `Binding`s.